package.json

```diff
@@ -49,9 +49,9 @@
     "babel-plugin-lodash": "^3.3.4",
     "babel-plugin-module-resolver": "^4.0.0",
     "babel-plugin-styled-components": "^1.7.1",
-    "babel-plugin-wildcard": "^6.0.0",
-    "cross-env": "^6.0.3",
-    "date-fns": "1.30.1",
+    "babel-plugin-wildcard": "^5.0.0",
```

Comment:
There some weird changes here on "babel-plugin-wildcard" & "cross-env".
Is it done on purpose ?

Reply:
No, I noticed that on the push, too. I just upgraded date-fns.